README.md

```diff
@@ -62,14 +62,16 @@ curl -L https://raw.githubusercontent.com/software-mansion/protostar/master/inst
    - To be able to build Protostar, set the following environmental variable before installing Python: `PYTHON_CONFIGURE_OPTS="--enable-shared"`
 1. Clone this repository
 1. Verify the active Python version: `python -V`
-1. Create Python virtual environment in the project directory: `python -m venv .venv`
-1. Activate environment: `source .venv/bin/activate`
-    - Consider using [direnv](https://direnv.net/) to activate the environment on navigating to the project directory
-1. Upgrade pip: `pip install --upgrade pip`
 1. [Install Poetry](https://python-poetry.org/docs/#installation) — a dependency manager
+    - Consider using `brew install poetry`
```

Comment:
```suggestion
```

This is not needed IMHO. Their blessed installation option should also work. What was bad is recommending using virtual environment for poetry.